src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,218 @@
+                PreparedStatement createCarsDriversStatement =
+                        connection.prepareStatement(query)) {
+            for (Driver driver: car.getDrivers()) {
+                createCarsDriversStatement.setLong(1, car.getId());
+                createCarsDriversStatement.setLong(2, driver.getId());
+                createCarsDriversStatement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't assign drivers for the car " + car, e);
+        }
+    }
+
+    private void updateDriversFor(Car car) {
+        String queryRemovePrevDrivers = "DELETE FROM cars_drivers WHERE id_car = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement removeDriversStatement =
+                        connection.prepareStatement(queryRemovePrevDrivers)) {
+            removeDriversStatement.setLong(1, car.getId());
+            removeDriversStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update drivers for the car " + car, e);
+        }
+        assignDriversToTheCar(car);
+    }
+
+    private Car getCar(ResultSet carResultSet) throws SQLException {
+        Long id = carResultSet.getObject("id", Long.class);
+        String model = carResultSet.getString(COLUMN_MODEL);
+        Long manufacturerId = carResultSet.getObject(COLUMN_MANUFACTURER_ID, Long.class);
+        String nameManufacturer = carResultSet.getString("name");
+        String countryManufacturer = carResultSet.getString("country");
+        Manufacturer manufacturer =
+                new Manufacturer(manufacturerId, nameManufacturer, countryManufacturer);
+        return new Car(id, model, manufacturer, new ArrayList<>());
+    }
+
+    private List<Driver> getAllDriversAssignedCar(Long id) {
+        String queryGetAllDriversAssignedCar =
+                "SELECT drivers.id, drivers.name, drivers.license_number FROM cars "
+                        + "JOIN cars_drivers ON cars.id = cars_drivers.id_car "
+                        + "JOIN drivers ON cars_drivers.id_driver = drivers.id "
+                        + "WHERE cars.id = ?";
+        List<Driver> result = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllCarDriversStatement =
+                        connection.prepareStatement(queryGetAllDriversAssignedCar)) {
+            getAllCarDriversStatement.setLong(1, id);
+            ResultSet driversIdResultSet = getAllCarDriversStatement.executeQuery();
+            while (driversIdResultSet.next()) {
+                Long driverId = driversIdResultSet.getLong("drivers.id");
+                String driverName = driversIdResultSet.getString("drivers.name");
+                String driverLicense = driversIdResultSet.getString("drivers.license_number");
```

Comment:
remove `drivers.` where you can + use alias if you have more than one columns with the same name